docs/usage.md

```diff
@@ -402,6 +402,67 @@ Other options can be used to help *git-changelog* retrieving
 the latest entry from your changelog: `--version-regex`
 and `--marker-line`.
 
+
+## Configuration files
+
+Project-wise, permanent configuration of *git-changelog* is possible.
+By default, *git-changelog* will search for the existence a suitable configuration
+in the ``pyproject.toml`` file or otherwise, the following configuration files 
+in this particular order:
+    * ``.git-changelog.toml``
+    * ``config/git-changelog.toml``
+    * ``.config/git-changelog.toml``
+    * ``~/.config/git-changelog.toml``
+
+The use of a configuration file can be disabled or overridden with the ``--config-file``
+option.
+To disable the configuration file, pass ``no``, ``None``, ``false``, or ``0``:
```

Comment:
None of them are for the CLI

Reply:
Not sure to understand your comment, but I saw in your code that values are case insensitive, so I think it would be better to document it as `none` here to avoid confusing it with Python's `None`.

```suggestion
To disable the configuration file, pass ``no``, ``none``, ``false``, or ``0``:
```

Reply:
Done, thanks!